CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Comment line immediately above ownership line is reserved for related gus information. Please be careful while editing.
```

Comment:
```suggestion
# Comment line immediately above ownership line is reserved for related information. Please be careful while editing.
```

Reply:
@wjhsf @ruoho -- I think we should just update the p/r with this change that removes GUS, and then accept. Does that sound good to you two?